Header Header
sensor_msgs/Imu imu
sensor_msgs/MagneticField mag_field
string IMU_raw_string
